wraith/logged_out.yaml

```diff
@@ -0,0 +1,202 @@
+  character_posts:
+    path: /characters/27?view=posts
+    selector: '[id=content]'
+  characters_search:
+    path: /characters/search
+    selector: '[id=content]'
+  characters_facecasts:
+    path: /characters/facecasts
+    selector: '[id=content]'
+
+  contribute:
+    path: /contribute
+    selector: '[id=content]'
+
+  galleries:
+    path: /users/3/galleries
+    selector: '[id=content]'
+  gallery_show:
+    path: /galleries/26
+    selector: '[id=content]'
+  galleries_search:
+    path: /galleries/search
+    selector: '[id=content]'
+
+  icon_show:
+    path: /icons/93
+    selector: '[id=content]'
+
+  posts:
+    path: /posts
+    before_capture: 'javascript/only-content.js'
+  post_show:
+    path: /posts/1
+    before_capture: 'javascript/only-content.js'
+  post_menu:
+    path: /posts/1
+    selector: '[id=post-menu-box]'
+  posts_search:
+    path: /posts/search
+    selector: '[id=content]'
+
+  replies_search:
+    path: /replies/search
+    selector: '[id=content]'
+
+  report_daily:
+    path: /reports/daily
+    selector: '[id=content]'
+
+  tags:
+    path: /tags
+    selector: '[id=content]'
+  tags_settings:
+    path: /tags?view=Setting
+    selector: '[id=content]'
+  tag_show:
+    path: /tags/2
+    selector: '[id=content]'
+
+  template_show:
+    path: /templates/11
+    selector: '[id=content]'
+  template_list:
+    path: /templates/11?view=list
+    selector: '[id=content]'
+  templates_search:
+    path: /templates/search
+    selector: '[id=content]'
+
+  users:
+    path: /users
+    selector: '[id=content]'
+  user_show:
+    path: /users/3
+    selector: '[id=content]'
+  user_boards:
+    path: /users/3/boards
+    selector: '[id=content]'
+  users_search:
+    path: /users/search
+    selector: '[id=content]'
+
+
+# (optional) JavaScript file to execute before taking screenshot of every path. Default: nil
+before_capture: 'javascript/accept_tos.js'
+
+# (required) Screen widths (and optional height) to resize the browser to before taking the screenshot.
+screen_widths:
+  - 750x1334
+  - 1024x768
+  - 1920x1080
+
+# (optional) Resize to each screen width (efficient), or reload at each screen width (costly). Default: 'reload'
+resize_or_reload: 'resize'
+
+# (required for history mode, otherwise optional) The directory that your base screenshots will be stored in.
+history_dir: 'then/out'
+
+# (required) The directory that your latest screenshots will be stored in
+directory: 'now/out'
+
+# (required) Amount of fuzz ImageMagick will use when comparing images. A higher fuzz makes the comparison less strict.
+fuzz: '20%'
+
+# (optional) The maximum acceptable level of difference (in %) between two images before Wraith reports a failure. Default: 0
+threshold: 5
+
+# (optional) Specify the template (and generated thumbnail sizes) for the gallery output.
+gallery:
+  template: 'slideshow_template' # Examples: 'basic_template' (default), 'slideshow_template'
+  thumb_width:  200
```

Comment:
Should these ben in core too?

Reply:
I think i looked and they're the defaults